app/Connection.php

```diff
@@ -8,6 +8,7 @@
 class Connection extends Model
 {
 
+    protected $fillable = ['user_id', 'owner_id'];
```

Comment:
Fuck fillable `protected $guarded = []`.

Mass assignment is a ridiculous protection that only saves idiots.